backend/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "body-parser": "^1.18.2",
     "cors": "^2.8.4",
+    "dotenv": "^6.0.0",
```

Comment:
I don't think this should be here. It should be installed globally. If you ran into issues using dotenv, just delete this line and you should still have it installed.